### DISCUSSION TITLE
Comfyui , ask for laptop recommendations

### DISCUSSION THREAD

**Discussion by maryf-t-s** [comfyanonymous/ComfyUI#6220] [Q&A]
Hi, I would like to install Comfyui on my local disk, but my computer doesn't meet the installation requirements. Can you recommend a laptop that can handle locally? Please suggest the most affordable options. From this description, it's hard for me to choose a laptop that would handle it. Maybe someone has a recommendation?

**Comment by LukeG89** [comment]
Well, first you have to consider two things:

1. Your budget: How much are you willing to spend on your setup?
2. What you want to do: Are you looking to generate images, videos, or something else?

ComfyUI is a pretty cool tool for AI-related stuff, but it needs some serious hardware to run demanding models. For example, if you've got a low-end laptop, you might be able to use some of the simpler models like SD1.5, but forget about running Flux.

Based on my own experience as a laptop user, here are some specs that'll help you choose:

* A decent NVIDIA GPU (I've got an RTX 3070 Ti and it's pretty good)
* At least 8GB of VRAM (16 GB or more is even better; trust me, VRAM is never enough 😅)
* A minimum of 32GB of RAM (64GB or more is ideal; my 64GB DDR5 saved me in many occasions)

And don't forget about CPU and SSD. My laptop's got an i7 12700H processor, and I've installed ComfyUI on an external 1TB SSD (Samsung T5) connected via USB-C 3.1.

So yeah, my setup's not the best, but it gets the job done in many scenarios (but it struggles a little with Flux and especially with video models like Hunyuan 😅). You can take this as reference.

But to be honest, I would suggest to consider building your own PC instead (if you can).

**Comment by CHNtentes** [comment]
If possible build a desktop PC instead. Laptops would suffer from throttling and would not provide same level of performance compared to similarly priced desktop.
